src/ofono/provider_ofono.cpp

```diff
@@ -972,12 +1020,12 @@ void MainNs::resetProperties(Bridge::Status status, SimPresent sim)
 
 #define PROP_(prop_name, value) { Bridge::name(Property::prop_name), value }
 
-MainNs::MainNs(QDBusConnection &bus, statefs_provider_mode mode)
-    : Namespace("Cellular", make_proper_source<Bridge>(mode, this, bus))
+MainNs::MainNs(QDBusConnection &bus, const char *name, ModemManager *manager, const QString &modem_pattern, statefs_provider_mode mode)
+    : Namespace(name, make_proper_source<Bridge>(mode, this, bus, manager, modem_pattern))
     , defaults_({
             PROP_(SignalStrength, "0")
                 , PROP_(DataTechnology, "unknown")
-                , PROP_(Status, "unregistered") // ofono
+                , PROP_(Status, "disabled") // ofono
```

Comment:
```// ofono``` comment should be removed: new "disabled" doesn't match any existing ofono status